settings.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+	// Gets URL of homepage
+	function homePageURL() {
+		$pageURL = 'http';
+		if (isset($_SERVER["HTTPS"])) 
+			$pageURL .= "s";
+		
+		$pageURL .= "://";
+		$pageURL .= $_SERVER["SERVER_NAME"];
+
+		return $pageURL;
+	}
+
+	// Get the inputs from the B2C Settings Page
+	$config_elements = get_option('b2c_config_elements');
```

Comment:
This is for the case where the user doesn't care to actually verify the id_tokens. The tokens will still be decoded, but the signature won't be checked.

---
In reply to: [71235652](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71235652) [](ancestors = 71235652)